bop_toolkit_lib/dataset_params.py

```diff
@@ -420,6 +415,18 @@ def get_split_params(datasets_path, dataset_name, split, split_type=None):
         p["quest3_im_size"] = {"rgb": (1408, 1408), "gray1": (640, 480), "gray2": (640, 480)}
         p["aria_im_size"] = {"gray1": (1280, 1024), "gray2": (1280, 1024)}
```

Comment:
are these correct? I think the names are swapped as well.

Reply:
indeed, will change